pkgs/unified_analytics/lib/src/log_handler.dart

```diff
@@ -67,6 +67,37 @@ class LogFileStats {
         'recordCount': recordCount,
         'eventCount': eventCount,
       });
+
+  /// Pass in a string label for one of the instance variables
+  /// and return the integer value of that label
+  ///
+  /// If label passed for [DateTime] instance variable, integer
+  /// in the form of [DateTime.millisecondsSinceEpoch] will be
+  /// returned
+  ///
+  /// Returns null if the label passed does not match anything
+  int? getValueByString(String label) {
+    switch (label) {
+      case 'logFileStats.startDateTime':
+        return startDateTime.millisecondsSinceEpoch;
```

Comment:
I think you should consider where to document these keys (I'm thinking specifically if some noogler in a few months writes a new survey condition where startDateTime < Jan 1 2023, they would need to know to put the condition in `millisecondsSinceEpoch`.

Could we add a test script to the website CI that validates the contextual-survey-metadata.json file is parseable by package:unified_analytics before publishing the website?

Reply:
Yeah that's a good point, I have added an additional item in the tracker issue to add automated checks for the metadata file in the `flutter/website` repo